src/readme-test.js

```diff
@@ -111,6 +111,36 @@ ${usageDocs}
       assert.calledWith(fs.writeFileSync, `${projectRoot}/README.md`, remarkResults);
     });
 
+    test('that consumer badges are lifted to top level when toc is present and no usage content', async () => {
```

Comment:
i'm wondering if calling out toc in this test is valuable since it doesnt directly impact the logic.

it's too bad that actually generating the toc is out of scope of the integration tests because that seems like the place to document that the placement of the zone relative to the toc at time of generation was the root issue with this bug

Reply:
yeah totally fair point. i can try to update it to better capture what the issue was.

Reply:
i've been thinking about this more since last night, but i don't feel like i've come up with a good solution for this. the logic here has nothing to do with toc, and i think that is fine and the test should reflect that. 

i wish there was a way to demonstrate the issue with the toc with a test at some scope, ideally in the integration suite, but there isnt.

this feels like a case where a comment near this test is the best solution. it is an issue that isnt related to the isolated logic, but it does influence _why_ the logic is the way it is